packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -291,14 +284,29 @@ const mathmode = ref('mathLIVE');
 // 	\beta IS - \gamma I \frac{d\color{red}{R}}{dt} = \gamma I`
 // };
 
+const betterStates = computed(() => {
```

Comment:
We have to agree between `State` and `Species`. Which one do you prefer @mwdchang ?

Reply:
I prefer `states`.